var/spack/repos/builtin/packages/mercury/package.py

```diff
@@ -9,21 +9,21 @@ class Mercury(CMakePackage):
     """Mercury is a C library for implementing RPC, optimized for HPC"""
 
     homepage = 'https://mercury-hpc.github.io/'
-    url = 'https://github.com/mercury-hpc/mercury/archive/0.9.0.tar.gz'
+    url = 'https://github.com/mercury-hpc/mercury/releases/download/v1.0.1/mercury-1.0.1.tar.bz2'
     git = 'https://github.com/mercury-hpc/mercury.git'
 
     version('develop', branch='master', submodules=True)
-    version('1.0.1', tag='v1.0.1', submodules=True)
-    version('1.0.0', tag='v1.0.0', submodules=True)
-    version('0.9.0', tag='v0.9.0', submodules=True)
-
-    variant('bmi', default=False, description='Use BMI for network transport')
-    variant('cci', default=False, description='Use CCI for network transport')
-    variant('mpi', default=False, description='Use MPI for network transport')
-    variant('ofi', default=True,  description='Use libfabric plugin')
+    version('1.0.1', preferred=True, sha256='02febd56c401ef7afa250caf28d012b37dee842bfde7ee16fcd2f741b9cf25b3')
```

Comment:
`preferred=True` isn't needed here, as Spack already defaults to the newest stable release.

Reply:
ok thanks, will fix that.